docker/ambulance-analytics-sandbox/src/routes/methodNotAllowed.js

```diff
@@ -0,0 +1,15 @@
+module.exports = [
+    /**
+     * GET to /submission
+     * If submissionResponse.json file exists in `mock` directory then that will be the response otherwise, 404
+     */
+    {
+      method: 'GET',
```

Comment:
Not a big issue but just for consistency with the `live` I think this will work better:
```
      method: ['GET', 'PUT', 'DELETE', 'PATCH'],
```